app/views/users/show.html.erb

```diff
@@ -3,25 +3,25 @@
         <img class="image-user" src=<%= @user.photo %> alt="user"/>
         <div class="user-info">
             <p class="user-name"><%= @user.name %></p>
-            <p class="user-post">Number of posts: <%= @user.posts_counter%></p> 
+            <p class="user-post">Number of posts: <%= @user.PostsCounter%></p> 
         </div>   
     </div>   
     <p class="user-bio"><%= @user.bio %></p>
 
     <% @user.last_3_posts.each do |post| %>
         <%= link_to user_post_path(@user.id, post.id) do %>
         <div class="user-recent-post">
-            <p class="post-title">Post : <%= post.title %></p>
+            <p class="post-title"><strong>Post</strong> : <%= post.title %></p>
             <p class="post-text"><%= post.text %></p>
-            <h3 class="counters">Comments: <%= post.comments_counter %>
-                Likes: <%= post.likes_counter %><h3>             
+            <h3 class="counters">Comments: <%= post.CommentSCounter %>
+                Likes: <%= post.LikesCounter %><h3>             
         </div>
```

Comment:
- [ ] Let's use the correct table column names here, your app doesn't have `PostsCounter`, `CommentsCounter`, or `LikesCounter`. Check your migration file or the schema for the correct posts table column names. Your app will work well if you fix these changes.

Reply:
![user](https://user-images.githubusercontent.com/53124548/218062273-f77289c2-4b3c-478c-9762-3c704b75d804.png)

Reply:
Thank you so much for the feedback, I have made the changes.